packages/components/src/dropdown-menu/style.scss

```diff
@@ -30,19 +30,15 @@
 			height: 1px;
 		}
 
-		&.is-active svg {
-			// Block UI appearance.
-			color: $white;
-			background: $gray-900;
-			box-shadow: 0 0 0 $border-width $gray-900;
-			border-radius: $border-width;
-		}
-
-		// Formatting buttons
-		> svg {
-			border-radius: $radius-block-ui;
-			width: $button-size-small;
-			height: $button-size-small;
```

Comment:
I don't think `border-radius` is necessary when the menu item is not active. Also, since the icon is guaranteed to be 24px in size, I don't think `width` and `height` are necessary.